api/api-doc/system.json

```diff
@@ -179,6 +179,21 @@
                ]
             }
          ]
+      },
+      {
+         "path":"/system/dump_profile",
```

Comment:
I think the word "profile" is overused - does it refer to performance profile (this is the normal meaning of the word "profiling")? Code coverage profile? Both? Neither?

The text below suggests it's about code coverage profile only, so can we call this dump_code_coverage (what does the word "profile" even mean?) to be clearer? Or if it's not just coverage, maybe change the text below?

Reply:
In llvm the PGO profile and the coverage profile are actually the same. A profile can be used either for profile
guided optimization or for producing code coverage data (or both :slightly_smiling_face: )

Reply:
I agree with @nyh that even though it's technically the same profile it is still ambiguous since the term "profile" is general. Perhaps dump_llvm_pgo_profile or dump_pgo_profile or dump_llvm_codecov_profile is a better name.

Reply:
I will change to dump_llvm_profile.
BTW, it can also be used for PGO, imagine the following scenario (which is actually described in the references of llvm).
You can have a workflow like that:
1. Start ScyllaDB
2. Run some scenario (for PGO purposes)
3. Dump the profile
4. Run another scenario (maybe with different hot paths)
5. Dump the profile again (it will be merged into the previous one)
6. repeat steps 4 and 5 as needed
7. terminate ScyllaDB
8. You now have a merged profile that represents multiple scenarios

And a final point about this, this is a dev api, a developer that uses it is set out to do something specific might it 
be PGO or coverage testing so I think that if this dev don't know what "profile" means he wouldn't try to use it in 
the first place.
But for having no doubt I will add a clarification in the help message.